src/test/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperationsTest.scala

```diff
@@ -184,8 +184,7 @@ class TezosPlatformDiscoveryOperationsTest
       }
 
       "return list of attributes of operations" in {
-        sut.getTableAttributes("operations").futureValue shouldBe
-          Some(
+        sut.getTableAttributes("operations").futureValue.get should contain allElementsOf
```

Comment:
I think `theSameElementsAs` would better fit the test.

`allElementsOf` means that the attributes might actually be _more than those listed_ in the expectations

Reply:
changed in all test cases and reformatted